chevah_build

```diff
@@ -39,6 +40,13 @@ EXTRA_LIBRARIES="\
    python-modules/python-setproctitle-1.1.8.dev0 \
    "
 
+# We only use pip on ArchLinux and Windows builds
```

Comment:
A better comment would be to describe how do you plan to use these variables... for ex;

```
# Space separate list of arguments sent to pip.
```

in this way someone managing them will know what type of value to put in these variables